### PR TITLE
atx.intro1.howard.matthew

### PR DESCRIPTION
**Pull request by matthew-s-howard**
add target attribute of blank

### CODE REVIEW

**Comment by e-zambrano** [comment]
@mhow8809 - nice work!

**Comment by e-zambrano** [comment]
@mhow8809 - excellent work!

File: 04LessonFour/in-class-project/css/style.css [hunk 1]
```diff
@@ -1,84 +1,26 @@
...
+}
+nav a {
+  color: white;
+}
+
+table, div {
+  border-collapse: collapse;
+}
+#mid {
+  clear: both;
```
Comment (e-zambrano) [inline 04LessonFour/in-class-project/css/style.css]: Not sure why you are using `clear:both`

File: 05LessonFive/Checkpoint1/index.html [hunk 2]
```diff
@@ -5,6 +5,27 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+    <header>
```
Comment (e-zambrano) [inline 05LessonFive/Checkpoint1/index.html]: Avoid adding a blank elements

File: 07LessonSeven/in-class-project/css/my-style.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  width: 250px;
+  margin-left: 20px;
+  vertical-align: top;
+  display: inline-block;
+
+}
+div:nth-child(2){
+  margin: 0 20px;
+}
+label {
```
Comment (e-zambrano) [inline 07LessonSeven/in-class-project/css/my-style.css]: if `label` and `input` have the same `declaration`, how can you combine the selectors to reduce redundancy?

File: 08LessonEight/in-class-project/homework.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+<body>
+    <header>
+        <h1>Recent From Blog</h1>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean molestie nulla eget odio hendrerit, vel faucibus ante facilisis. Cras eget nulla elit.
+        </p>
+    </header>
+    <main class="clearfix">
+        <section id="">
+            <img src="img/keyboard.jpg" />
```
Comment (e-zambrano) [inline 08LessonEight/in-class-project/homework.html]: I can't see the image. Notice how the extension is `JPG` and not `jpg`

File: 08LessonEight/in-class-project/css/homework-style.css [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  line-height:14px;
+}
+header {
+  text-align: center;
+  width: 400px;
+  margin: 0 auto;
+}
+main {
+
+    margin: 0 auto;
```
Comment (e-zambrano) [inline 08LessonEight/in-class-project/css/homework-style.css]: If i remove this entire CSS rule it wont change a thing. What where you trying to do here?

File: 08LessonEight/homework/css/exercise-06-style.css [hunk 6]
```diff
@@ -1 +1,62 @@
...
+/*Add CSS rules*/
+body {
+  width: 1000px;
+  margin: 0 auto;
+  padding: 40px 0;
+  font-family: "Arial", sans-serif;
+  font-size: 12px;
+  line-height: 16px;
+}
+section {
```
Comment (e-zambrano) [inline 08LessonEight/homework/css/exercise-06-style.css:10]: Remove empty selector

File: 08LessonEight/homework/css/exercise-06-style.css [hunk 7]
```diff
@@ -1 +1,62 @@
...
+
+}
+.clearfix: after {
```
Comment (e-zambrano) [inline 08LessonEight/homework/css/exercise-06-style.css]: remove space between `:` and `after`

File: 08LessonEight/homework/css/exercise-06-style.css [hunk 8]
```diff
@@ -1 +1,62 @@
...
+a {
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: 10px;
+  border: 1px solid lightgrey;
+  border-right: 3px solid red;
+  color: #000;
+}
+
+#truck .image {
```
Comment (e-zambrano) [inline 08LessonEight/homework/css/exercise-06-style.css]: Noticed how your `#truck .image`, `#plane .image`, and `#train .image` have the same properties. How can you improve that and reduce redundancy?

**Comment by e-zambrano** [comment]
@m-how2, a few things:
- Avoid capitalizing ids or classes selectors ex. ".My-class {...}"
- Make sure to beautify your code before committing

You are doing a great job!

File: 13LessonThirteen/homework/css/style.css [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+#hills {
+  background-image: url("http://bone.themanoid.com/images/thumbs/07.png");
+}
+#devices {
+  background-image: url("http://bone.themanoid.com/images/thumbs/02.png");
+}
+#rockface {
+  background-image: url("http://bone.themanoid.com/images/thumbs/04.png");
+}
+#building {
```
Comment (zacacollier) [inline 13LessonThirteen/homework/css/style.css:29]: FYI the dimensions of these images is inconsistent when the browser is sized to medium screen width - they should all be the same height and width

File: 13LessonThirteen/homework/css/style.css [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+    font-size: 2.5em;
+    font-family: Roboto Mono,Helvetica Neue,sans-serif;
+    font-weight: 200;
+  }
+  .items h3 {
+    font-size: 1.7em;
+    margin-bottom: 25px;
+  }
+  .items h4 {
+    font-size: 2em;
```
Comment (zacacollier) [inline 13LessonThirteen/homework/css/style.css:96]: try to be consistent with your units of measurement - IMO best practice is to just stick with `px`

File: 13LessonThirteen/homework/index.html [hunk 11]
```diff
@@ -1,10 +1,154 @@
...
+            </div>
+        </div>
+    </section>
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <!-- Include all compiled plugins (below), or include individual files as needed -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
 </body>
+
 </html>
```
Comment (zacacollier) [inline 13LessonThirteen/homework/index.html:154]: Well done!
